reference/recency-surveillance.v1.json

```diff
@@ -359,7 +364,7 @@
                           "Concordant Positive"
                         ]
                       },
-                      "presumtive_tb": {
+                      "presumptive_tb": {
```

Comment:
This should not be corrected. You can correct the spelling in the description, but not a column name spelling. This would mean all the previous implementations need to be updated.

Reply:
@davidmugume The issue has been corrected.

Reply:
I still see it.

Reply:
I have applied changes from the most recent commit and it resolves issue:
https://github.com/METS-Programme/hie-documentation/pull/8/files

Reply:
> This should not be corrected. You can correct the spelling in the description, but not a column name spelling. This would mean all the previous implementations need to be updated.

Ooh disregard my request change based on this comment

Reply:
@davidmugume  I can only find Musoke, Kaye and Odora for reviewers on this repo, Samuel is not listed.